api/docker/caddy/Caddyfile

```diff
@@ -34,17 +34,6 @@
 }
 
 :3001 {
-	# Matches requests for HTML documents, for static files,
```

Comment:
PWA is still there in the referenced https://github.com/api-platform/api-platform/compare/d059b22b4e3b4e2a6e30d7ca3c2921145a367e3f...7642f8639ff1d95cfa48832db0221973559f8d29
Is this only used for their admin application?

Reply:
Yes, they use it for their admin application. And it seems they also use NextJS to serve other static files.

This removed code is only the definition of the `@pwa expression` and doesn't change any behavior of Caddy alone. In the api-platform repo, this expression is reference[ further below in the reverse_proxy statement](https://github.com/api-platform/api-platform/blob/bdf8b894d53b6c640e6f60e8969087433d5aac1d/api/docker/caddy/Caddyfile#L46). We don't make use of this expression anywhere, so there's no reason in keeping it in our file. Unless we want to use similar expressions instead of our `handle` and `handle_path` directives. Don't know enough about Caddy though to know the difference between using request matchers and handle directives.